src/components/jumbotron/jumbotron.js

```diff
@@ -84,7 +84,7 @@ export const BJumbotron = /*#__PURE__*/ Vue.extend({
               [`display-${props.headerLevel}`]: props.headerLevel
             }
           },
-          normalizeSlot('header', {}, $scopedSlots, $slots) ||
+          normalizeSlot('header', $scopedSlots, $slots) ||
```

Comment:
`{}` is the scope for normalizeSlot... it can't be skipped like `h(tag, {}, ...)` can.
So what is now happening is that it is seeing `$scopedSlots` as the scope to pass to the function, and seeing `$slots` as the `$scopedSlots` function.

Reply:
Good catch, this is a mistake.